indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -331,6 +333,26 @@ def _process(self):
         return send_file('registrations.csv', csv_file, 'text/csv')
 
 
+class RHRegistrationsPrintBadges(RHRegistrationsActionBase):
+    """Print badges for the selected registrations"""
+
+    def _process(self):
+        badge_templates = self.event.getBadgeTemplateManager().getTemplates().items()
+        badge_templates.sort(lambda item1, item2: cmp(item1[1].getName(), item2[1].getName()))
+        template_list = render_template('events/registration/management/_badge_template.html',
+                                        templates=badge_templates)
+        registration_list = render_template('events/registration/management/_badges_registration_list.html',
```

Comment:
you could `{% include %}` that template (or even better use a macro and import/call it) instead of rendering it here